PwaDigitalGoods/explainer.md

```diff
@@ -65,7 +65,11 @@ Combined with the Payment Request API, users can purchase digital products from
 
 ## Usage Examples
 
-The Digital Goods API allows the user agent to provide the above operations, alongside digital store integration via the Payment Request API.
+The Digital Goods API allows the user agent to provide the above operations, alongside digital store integration via the Payment Request API. 
+
+
+> **IMPORTANT**
+If you want to try this API with Microsoft Store, please use the latest Edge Beta/Dev/Canary. The Edge Stable currently does not support Digital Goods API on Windows.
```

Comment:
Since this information will get out of date quickly, can you specify the exact version of Edge instead?

```suggestion
If you want to try this API with Microsoft Store, please use the Edge version X.X.X.X or greater.
```

Reply:
Modified.